iothub/service/src/Utilities/Extensions/HttpHeadersExtensions.cs

```diff
@@ -1,17 +1,16 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Net.Http.Headers;
 
-namespace Microsoft.Azure.Devices.Utilities
```

Comment:
Folders give us a way to organize files, but we don't want to pay the cost of including lots of extra namespaces.